CHANGELOG.md

```diff
@@ -77,8 +77,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Fixed
 
-- Bug Fix - Form kit dependencies ([#481][] @terryfinn)
+- Bug Fix - Null class concatenation on Card Kit ([#497][] @jasperfurniss)
```

Comment:
Nice! Please move to the `Unreleased` section under `### Fixed` category (you can create it there)? :)

Reply:
Also please add new items below current ones, that facilitates understanding what you added (you would only have green additions instead of red and green).

Also please link to the PR by adding the following to the respective section you're adding the changelog entry:

```
[#497]: https://github.com/powerhome/playbook/pull/497
```